src/css/mapbox-gl.css

```diff
@@ -440,35 +441,30 @@ a.mapboxgl-ctrl-logo.mapboxgl-compact {
 @media screen {
     .mapboxgl-ctrl-attrib.mapboxgl-compact {
         min-height: 20px;
-        padding: 0;
+        padding: 2px 24px 2px 0;
         margin: 10px;
         position: relative;
         background-color: #fff;
-        border-radius: 3px 12px 12px 3px;
+        border-radius: 12px;
     }
 
-    .mapboxgl-ctrl-attrib.mapboxgl-compact:hover {
-        padding: 2px 24px 2px 4px;
+    .mapboxgl-ctrl-attrib.mapboxgl-compact-show {
+        padding: 2px 24px 2px 8px;
         visibility: visible;
-        margin-top: 6px;
     }
 
-    .mapboxgl-ctrl-top-left > .mapboxgl-ctrl-attrib.mapboxgl-compact:hover,
-    .mapboxgl-ctrl-bottom-left > .mapboxgl-ctrl-attrib.mapboxgl-compact:hover {
-        padding: 2px 4px 2px 24px;
-        border-radius: 12px 3px 3px 12px;
+    .mapboxgl-ctrl-top-left > .mapboxgl-ctrl-attrib.mapboxgl-compact-show,
+    .mapboxgl-ctrl-bottom-left > .mapboxgl-ctrl-attrib.mapboxgl-compact-show {
+        padding: 2px 8px 2px 24px;
+        border-radius: 12px;
     }
 
     .mapboxgl-ctrl-attrib.mapboxgl-compact .mapboxgl-ctrl-attrib-inner {
         display: none;
     }
 
-    .mapboxgl-ctrl-attrib.mapboxgl-compact:hover .mapboxgl-ctrl-attrib-inner {
-        display: block;
-    }
-
-    .mapboxgl-ctrl-attrib.mapboxgl-compact::after {
-        content: '';
+    .mapboxgl-ctrl-attrib-button {
```

Comment:
<img width="434" alt="Screen Shot 2020-10-01 at 11 44 03 AM" src="https://user-images.githubusercontent.com/4523080/94851513-6873b000-03dd-11eb-97ef-14f18e002796.png">

I think the compact attribution could use a bit more space between the text and the button. When the button is focused, which it is when the control is first maximized after clicking it or hitting Enter, the blue focus ring abuts the text pretty tightly.

Reply:
Increased padding in https://github.com/mapbox/mapbox-gl-js/pull/9991/commits/67282f40d3cef6a69b4f548ae2163c4df69cb45d to address this!

<img width="327" alt="Screen Shot 2020-10-04 at 1 27 37 PM" src="https://user-images.githubusercontent.com/61150/95022444-60b64480-0645-11eb-80cc-ebb85e924d46.png">